core/src/com/po/fuck/model/weapons/CosmicBullet.java

```diff
@@ -14,8 +14,9 @@ public final class CosmicBullet extends Bullet {
         damage = COSMIC_BULLET_DAMAGE;
         lifeTime = COSMIC_BULLET_LIFE_TIME;
         geometryData = new GeometryData();
-        geometryData.setWidth(new Sprite(new Texture("bullet2.png")).getWidth());
-        geometryData.setHeight(new Sprite(new Texture("bullet2.png")).getHeight());
+        BasicSpriteInfo thisInfo = BasicSpriteManager.getBasicSpriteInfo(this.getClass());
+        geometryData.setHeight(thisInfo.getHeight());
+        geometryData.setWidth(thisInfo.getWidth());
```

Comment:
Rename `thisInfo` -> `info`. Change the code to `geometryData.setSize(info.getSize())`